test/functional/content_view/version/update_test.rb

```diff
@@ -0,0 +1,97 @@
+require_relative '../../test_helper'
+require 'hammer_cli_katello/content_view_version'
+
+module HammerCLIKatello
+  describe ContentViewVersion::UpdateCommand do
+    include OrganizationHelpers
+    it 'allows minimal options' do
+      ex = api_expects(:content_view_versions, :update).with_params('description' => 'pizza')
+      ex.returns('id' => '2', 'description' => 'pizza')
+
+      result = run_cmd(%w(content-view version update --id 2 --description pizza))
+      assert_equal(result.exit_code, 0)
+    end
+  end
+
+  describe 'content view options' do
+    it 'allows content view id' do
+      ex = api_expects(:content_view_versions, :update).with_params('description' => 'pizza')
+      ex.returns('id' => '2', 'description' => 'pizza', 'content_view' => [{'id' => '2'}])
+
+      # rubocop:disable LineLength
+      result = run_cmd(%w(content-view version update --id 2 --description pizza --content-view-id 2))
+      # rubocop:enable LineLength
+      assert_equal(result.exit_code, 0)
+    end
+
+    it 'allows content view name' do
+      ex = api_expects(:content_view_versions, :update).with_params('description' => 'pizza')
+      ex.returns('id' => '2', 'description' => 'pizza', 'content_view' => [{'name' => 'cv1'}])
+
+      # rubocop:disable LineLength
+      result = run_cmd(%w(content-view version update --id 2 --description pizza --content-view cv1))
```

Comment:
This one should require `--organization-id` because you've provided `--content-view` (name). Same thing as above though, you should be using `--id` with `--content-view[-id]`